examples/planning/python/simple_client.py

```diff
@@ -0,0 +1,270 @@
+import grpc
+import sys
+import yaml
+import os
+import os.path
+from typing import Tuple
+
+ROOT_DIR = os.path.dirname(os.path.abspath(__file__))
+DATA_DIR = os.path.join(os.path.dirname(ROOT_DIR), "data")
+BUILDFILES_DIR = os.path.join(ROOT_DIR, "build")
+if BUILDFILES_DIR not in sys.path:
+    sys.path.append(BUILDFILES_DIR)
+
+from build.types_pb2 import (
+    Constraints,
+    Model,
+    SystemConf,
+    Conf,
+    PositionConstraint,
+    AngleBetweenVectorsConstraint,
+    PlanContextId,
+)
+from build.generate_id_pb2 import (
+    RegisterPlanContextRequest,
+    RegisterPlanContextResponse,
+)
+from build.generate_id_pb2_grpc import PlanContextRegistryStub
+from build.builder_pb2 import StartBuildFromConfsRequest, StartBuildResponse
+from build.builder_pb2_grpc import IrisBuilderStub
+
+
+def make_position_constraint_msg(
+    frame_A: str,
+    frame_B: str,
+    p_AQ_lower: Tuple[float, float, float],
+    p_AQ_upper: Tuple[float, float, float],
+    position_BQ: Tuple[float, float, float],
+) -> PositionConstraint:
+    """Make a Protobuf message for a position constraint on a point Q atached
```

Comment:
Is it possible to put this doc in the protobuf api? I honestly don't know the answer to that question.

Reply:
Yeah, that's fine.